src/utils/apis/LoginAPI.js

```diff
@@ -38,10 +38,14 @@ export const loginAPI = {
     },
 
     registerCompany: function(login, password, name) {
-        return new Promise(resolve => {
-            setTimeout(() => {
-                resolve({text: "ok", ok: true})
-            }, 2000)
+        return fetch(recruitmentServiceBasicAPILink + "/api/organizations", {
+            method: "POST",
+            headers: headers,
+            body: JSON.stringify({
+                name: name,
```

Comment:
Ogólnie protip na przyszłość. Jak w obiekcie jest taki sam label jak nazwa zmiennej (jak tutaj jest name i name), to label można pominąć i zostanie tylko jeden name. Ale jak mówię to protip na przyszłość, tu można zostawić.